docker-compose up -d --build

By following this doc: https://github.com/monicahq/docker/tree/main/.examples.
And I use command: docker-compose up -d --build.
And I got this:
```
146.3 checking for libmemcached location... configure: error: memcached support requires libmemcached. Use --with-libmemcached-dir=<DIR> to specify the prefix where libmemcached headers and library are located
146.4 ERROR: `/tmp/pear/temp/memcached/configure --with-php-config=/usr/local/bin/php-config --with-libmemcached-dir=no --with-zlib-dir=no --with-system-fastlz=no --enable-memcached-igbinary=no --enable-memcached-msgpack=no --enable-memcached-json=no --enable-memcached-protocol=no --enable-memcached-sasl=yes --enable-memcached-session=yes' failed
------
failed to solve: executor failed running [/bin/sh -c set -ex;         savedAptMark="$(apt-mark showmanual)";         apt-get update;     apt-get install -y --no-install-recommends         libicu-dev         zlib1g-dev         libzip-dev         libpng-dev         libxml2-dev         libfreetype6-dev         libjpeg62-turbo-dev         libgmp-dev         libmemcached-dev         libmagickwand-dev         libwebp-dev     ;         debMultiarch="$(dpkg-architecture --query DEB_BUILD_MULTIARCH)";     if [ ! -e /usr/include/gmp.h ]; then ln -s /usr/include/$debMultiarch/gmp.h /usr/include/gmp.h; fi;    docker-php-ext-configure intl;     docker-php-ext-configure gd --with-jpeg --with-freetype --with-webp;     docker-php-ext-configure gmp;     docker-php-ext-install -j$(nproc)         intl         zip         bcmath         gd         gmp         pdo_mysql         mysqli         soap     ;         pecl install APCu-5.1.22;     pecl install memcached-3.2.0RC2;     pecl install redis-5.3.7;         docker-php-ext-enable         apcu         memcached         redis     ;         apt-mark auto '.*' > /dev/null;     apt-mark manual $savedAptMark;         ldd "$(php -r 'echo ini_get("extension_dir");')"/*.so         | awk '/=>/ { print $3 }'         | sort -u         | xargs -r dpkg-query -S         | cut -d: -f1         | sort -u         | xargs -rt apt-mark manual;             apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false;     rm -rf /var/lib/apt/lists/*]: exit code: 1

```

If I add `RUN apt-get update; apt-get install -y libmemcached-dev zlib1g-dev libssl-dev` , I got this:
```
 => ERROR [app 15/17] RUN set -ex;         mkdir -p storage/framework/views;     composer install --no-interaction --no-progress --no-dev;     composer clear-cache;     rm -rf .composer                2.7s
------
 > [app 15/17] RUN set -ex;         mkdir -p storage/framework/views;     composer install --no-interaction --no-progress --no-dev;     composer clear-cache;     rm -rf .composer:
0.858 + mkdir -p storage/framework/views
0.937 + composer install --no-interaction --no-progress --no-dev
1.790
1.790 Warning: PHP Startup: Unable to load dynamic library 'gd.so' (tried: /usr/local/lib/php/extensions/no-debug-non-zts-20210902/gd.so (libpng16.so.16: cannot open shared object file: No such file or directory), /usr/local/lib/php/extensions/no-debug-non-zts-20210902/gd.so.so (/usr/local/lib/php/extensions/no-debug-non-zts-20210902/gd.so.so: cannot open shared object file: No such file or directory)) in Unknown on line 0
1.790
1.790 Warning: PHP Startup: Unable to load dynamic library 'zip.so' (tried: /usr/local/lib/php/extensions/no-debug-non-zts-20210902/zip.so (libzip.so.4: cannot open shared object file: No such file or directory), /usr/local/lib/php/extensions/no-debug-non-zts-20210902/zip.so.so (/usr/local/lib/php/extensions/no-debug-non-zts-20210902/zip.so.so: cannot open shared object file: No such file or directory)) in Unknown on line 0
2.017 Composer plugins have been disabled for safety in this non-interactive session. Set COMPOSER_ALLOW_SUPERUSER=1 if you want to allow plugins to run as root/super user.
2.018 Do not run Composer as root/super user! See https://getcomposer.org/root for details
2.152 > App\Helpers\ComposerScripts::preInstall
2.165 Installing dependencies from lock file
2.176 Verifying lock file contents can be installed on current platform.
2.233 Your lock file does not contain a compatible set of packages. Please run composer update.
2.233
2.233   Problem 1
2.233     - Root composer.json requires PHP extension ext-gd * but it is missing from your system. Install or enable PHP's gd extension.
2.233
2.233 To enable extensions, verify that they are enabled in your .ini files:
2.233     - /usr/local/etc/php/conf.d/docker-php-ext-apcu.ini
2.233     - /usr/local/etc/php/conf.d/docker-php-ext-bcmath.ini
2.233     - /usr/local/etc/php/conf.d/docker-php-ext-gd.ini
2.233     - /usr/local/etc/php/conf.d/docker-php-ext-gmp.ini
2.233     - /usr/local/etc/php/conf.d/docker-php-ext-intl.ini
2.233     - /usr/local/etc/php/conf.d/docker-php-ext-memcached.ini
2.233     - /usr/local/etc/php/conf.d/docker-php-ext-mysqli.ini
2.233     - /usr/local/etc/php/conf.d/docker-php-ext-opcache.ini
2.233     - /usr/local/etc/php/conf.d/docker-php-ext-pdo_mysql.ini
2.233     - /usr/local/etc/php/conf.d/docker-php-ext-redis.ini
2.233     - /usr/local/etc/php/conf.d/docker-php-ext-soap.ini
2.233     - /usr/local/etc/php/conf.d/docker-php-ext-sodium.ini
2.233     - /usr/local/etc/php/conf.d/docker-php-ext-zip.ini
2.233     - /usr/local/etc/php/conf.d/memory-limit.ini
2.233     - /usr/local/etc/php/conf.d/opcache-recommended.ini
2.233 You can also run `php --ini` in a terminal to see which files are used by PHP in CLI mode.
2.233 Alternatively, you can run Composer with `--ignore-platform-req=ext-gd` to temporarily ignore these required extensions.
------
failed to solve: executor failed running [/bin/sh -c set -ex;         mkdir -p storage/framework/views;     composer install --no-interaction --no-progress --no-dev;     composer clear-cache;     rm -rf .composer]: exit code: 2

```

Please give me some advice, thank you.

--- a/scripts/docker/Dockerfile
+++ b/scripts/docker/Dockerfile
@@ -3,7 +3,7 @@
 ###
 ### This file is used for dev purpose.
 ### The standard monica image definition will be found here: https://github.com/monicahq/docker
-### This file is based off of the `apache` variant in the above mentioned repo
+### This file is based on the `apache` variant in the above mentioned repo
 ###
 
 ## Build assets
@@ -13,7 +13,7 @@ WORKDIR /var/www/html
 COPY . ./
 RUN set -ex; \
     \
-    yarn install; \
+    yarn install --immutable; \
     yarn run build
 
 
@@ -48,8 +48,9 @@ RUN set -ex; \
         libjpeg62-turbo-dev \
         libgmp-dev \
         libmemcached-dev \
-        libmagickwand-dev \
+        libssl-dev \
         libwebp-dev \
+        libcurl4-openssl-dev \
     ; \
     \
     debMultiarch="$(dpkg-architecture --query DEB_BUILD_MULTIARCH)"; \
@@ -99,7 +100,7 @@ RUN set -ex; \
     \
     mkdir -p /var/spool/cron/crontabs; \
     rm -f /var/spool/cron/crontabs/root; \
-    echo '*/5 * * * * php /var/www/html/artisan schedule:run -v' > /var/spool/cron/crontabs/www-data
+    echo '* * * * * php /var/www/html/artisan schedule:run -v' > /var/spool/cron/crontabs/www-data
 
 # Opcache
 ENV PHP_OPCACHE_VALIDATE_TIMESTAMPS="0" \
@@ -158,7 +159,6 @@ RUN set -ex; \
     sed -ri -e "s!/var/www/!${APACHE_DOCUMENT_ROOT}!g" $APACHE_CONFDIR/apache2.conf $APACHE_CONFDIR/conf-available/*.conf
 
 WORKDIR /var/www/html
-
 
 # Copy the local (outside Docker) source into the working directory,
 # copy system files into their proper homes, and set file ownership

